ts_benchmark/evaluation/strategy/rolling_forecast.py

```diff
@@ -260,13 +260,17 @@ def _eval_sample(
             len(index_list), num_rollings
         )
         single_series_results = np.mean(np.stack(all_test_results), axis=0).tolist()
-        # we do not save rolling results by default because it is often too large
+
+        save_true_pred = self._get_scalar_config_value("save_true_pred", series_name)
```

Comment:
is it necessary to add 'save_true_pred' to the REQUIRED_CONFIGS variable?

Reply:
it has been added